extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -13,3 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-autocomplete {
+  position: relative;
+  display: inline-block;
+}
+
+.i-amphtml-autocomplete-input {
+  padding: 4px;
+}
+
+.i-amphtml-autocomplete-results {
+  position: absolute;
+  top: 100%;
```

Comment:
I think `bottom:0` may work the same and if so, it would be preferable since `top:100%` would be dependent on paddings, border and boxing properties that user can configure.

Reply:
It looks like `top: 100%` positions the results below the input element and `bottom: 0` positions the results and the input element in the same vertical location/overlapping. Is there a way to use  `bottom: 0` and still account for the height of the input element?

Reply:
not sure, if `bottom: 0` not working, let's keep what you have